interfaces/udev/spec.go

```diff
@@ -94,7 +94,7 @@ func (spec *Specification) TagDevice(snippet string) {
 	for _, securityTag := range spec.securityTags {
 		tag := udevTag(securityTag)
 		spec.addEntry(fmt.Sprintf("# %s\n%s, TAG+=\"%s\"", spec.iface, snippet, tag), tag)
-		spec.addEntry(fmt.Sprintf("TAG==\"%s\", RUN+=\"%s/snap-device-helper $env{ACTION} %s $devpath $major:$minor\"",
+		spec.addEntry(fmt.Sprintf("TAG==\"%s\", SUBSYSTEM!=\"module\", SUBSYSTEM!=\"subsystem\", RUN+=\"%s/snap-device-helper $env{ACTION} %s $devpath $major:$minor\"",
```

Comment:
Perhaps an explanation of why?

Reply:
I agree, a comment explaining the SUBSYSTEM rules would be good for the next person that touches this code

Reply:
I have added some comments.

Reply:
Thanks for adding this! It is much easier to follow now for me :)